src/Microsoft.Health.Fhir.Anonymizer.R4.CommandLineTool/configuration-sample.json

```diff
@@ -1,5 +1,6 @@
 {
   "fhirVersion": "R4",
+  "processingErrors": "raise",
```

Comment:
It seems our config file is case insensitive right?

Reply:
Yes